Sources/NIOCore/ChannelOption.swift

```diff
@@ -287,6 +287,13 @@ extension ChannelOptions {
             public typealias Value = Bool
             public init() {}
         }
+        
+        /// `BufferedWritableBytesOption` allows users to know the number of writable bytes currently buffered in the `Channel`.
+        public struct BufferedWritableBytesOption: ChannelOption, Sendable {
+            public typealias Value = Int64
```

Comment:
The preference in Swift is to use `Int` at API boundaries such as this. Given that a buffer size of more than 2GB on a 32-bit system is going to cause A Bad Time anyway, we can safely make that change here.